lightly/api/api_workflow_sampling.py

```diff
@@ -0,0 +1,93 @@
+import time
+from typing import *
+
+from lightly.active_learning.config.sampler_config import SamplerConfig
+from lightly.openapi_generated.swagger_client.models.job_state import JobState
+from lightly.openapi_generated.swagger_client.models.job_status_data import JobStatusData
+from lightly.openapi_generated.swagger_client.models.tag_data import TagData
+from lightly.openapi_generated.swagger_client.models.sampling_config import SamplingConfig
+from lightly.openapi_generated.swagger_client.models.sampling_create_request import SamplingCreateRequest
+from lightly.openapi_generated.swagger_client.models.sampling_config_stopping_condition import \
+    SamplingConfigStoppingCondition
+
+
+class _SamplingMixin:
+    def sampling(self, sampler_config: SamplerConfig, al_scores: Dict[str, List[int]] = None,
+                 preselected_tag_id: str = None, query_tag_id: str = None) -> TagData:
+        """Performs a sampling given the arguments.
+
+        Args:
+            sampler_config:
+                The configuration of the sampler.
+            al_scores:
+                The active learning scores for the sampler.
+            preselected_tag_id:
+                The tag defining the already chosen samples (e.g. already labelled ones), default: None.
+            query_tag_id:
+                The tag defining where to sample from, default: None resolves to the initial-tag.
+
+        Returns:
+            The newly created tag of the sampling.
+
+        Raises:
+            ApiException, ValueError
+        """
+
+        # upload the active learning scores to the api
+        if al_scores is not None:
+            raise NotImplementedError  # TODO: fill out in later branches
+
+        # trigger the sampling
+        payload = self._create_sampling_create_request(sampler_config,preselected_tag_id,query_tag_id)
+        payload.row_count = 15  # TODO: remove after solving https://github.com/lightly-ai/lightly-core/issues/150
```

Comment:
So now we can only use it on this specific test case? Can we not determine the row count by the initial-tag size? Would make it much much easier to test